### PR TITLE
Add Thorium docs

### PR DESCRIPTION
**Pull request by thatch45**

### CODE REVIEW

File: doc/topics/releases/2016.3.0.rst [hunk 1]
```diff
@@ -80,6 +80,21 @@ Platform Changes
 New Features
 ============
 
+Thorium - Provisional New Reactor
+---------------------------------
+
+The 2016.3 release introduces the new Thorium Reactor. This reactor is an
+experimental new feature that impliments a flow programing interface using
```
Comment (s0undt3ch) [inline doc/topics/releases/2016.3.0.rst]: Implements?

**Comment by thatch45** [comment]
lol, I just nuked my laptop, back to Arch, and I had not installed hunspell-en so my spellchecked was lying to me :)

**Comment by s0undt3ch** [comment]
Everyone hates a lying spell checker :)

**Comment by jacobhammons** [comment]
Added to release notes, removing label.

**Pull request merged**